### ISSUE TITLE
Problem: No fiat/USD pricing is displayed on bid page

### ISSUE THREAD

**Issue by alexanderdushkin** [ReinProject/python-rein#89]
When a user bids, he doesn't get to know how much his bid is worth in USD or any other currency. 

It would be convenient to let a user choose the currency he needs on setup or in settings, and then display a second field on the bid page with the calculated amount.

**Comment by weex** [comment]
I agree this would be a nice-to-have along with some redundancy/averaging in which pricing APIs are used.
